config/manager/kustomization.yaml

```diff
@@ -0,0 +1,16 @@
+resources:
```

Comment:
should not need checkin this temp-generated file

Reply:
right! I removed this file

Reply:
I guess we need to remove it again... which brings back my point from https://github.com/opendatahub-io/opendatahub-operator/pull/728#issuecomment-1921023371

Reply:
the last piece and we are ready to go